hosting/src/app/(protected)/error/page.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+import Loader from "@/components/common/Loader";
+import Notification from "@/components/common/Notification";
+import PageHeader from "@/components/common/PageHeader";
+import {useAuthentication} from "@/hooks/useAuthentication";
+import {useTanamUser} from "@/hooks/useTanamUser";
+import {Suspense} from "react";
+
+export default function ErrorPage() {
+  const {authUser} = useAuthentication();
+  const {error: userError} = useTanamUser(authUser?.uid);
```

Comment:
The component name is `ErrorPage` but it only supports user errors. Seems misleading.

Make this into a generic component that takes in error messages to display or rename the component to be specific for the error that it displays.

Reply:
Done ✅ 
I created specific error page for this cases